src/extension/content-script/batteries/VimeoVideo.ts

```diff
@@ -0,0 +1,47 @@
+import getOriginData from "../originData";
+import setLightningData from "../setLightningData";
+
+const urlMatcher = /^https:\/\/vimeo.com\/.*/;
```

Comment:
I think we can limit this a bit down.
seems all vimeo video pages end with an ID.
https://vimeo.com/584985260 or https://vimeo.com/channels/bestoftheyear2021/584985260

maybe: `^https:\/\/vimeo.com\/.*\d{4,}\/?\z`

that it must end at least with a 4 digits number?

Reply:
I couldn't get the one you have working I think due to the `\z` at the end.
but made this one:
^https:\/\/vimeo.com\/.*(\d+)(?:|\/\?)
link: https://regexr.com/6e6au
yours does seem simpler and better if I can understand the `\z`

Reply:
maybe it should be `$` instead of `\z` - I just thought to check that it ends with a number

Reply:
that did it because with `\z` eslinter was complaining.

Reply:
ah ja. great. 
that regexr.com site is nice. didn't know it.